stackstate-deps.json

```diff
@@ -1,5 +1,5 @@
 {
-  "STACKSTATE_INTEGRATIONS_VERSION": "1.15.0",
+  "STACKSTATE_INTEGRATIONS_VERSION": "master",
```

Comment:
Not too relevant right now, seeing as we'll do a release soon. But we shouldn't target master of the agent integrations in the agent repo. It's typically better to make a stable tag ("release") of the agent integrations and target that. We can do multiple tags of integrations per agent release, so we shouldn't tie them too closely.